lib/Dialect/HW/HWOps.cpp

```diff
@@ -754,16 +754,12 @@ Operation *InstanceOp::getReferencedModule(const SymbolCache *cache) {
       return result;
 
   auto topLevelModuleOp = (*this)->getParentOfType<ModuleOp>();
-  if (!topLevelModuleOp)
```

Comment:
This is safe now that I just landed the instance labels change, but please make this an assertion so we get a good message on malformed IR.

Reply:
If I understand correctly, this is guaranteed by the HasParent trait of `InstanceOp`.

Reply:
https://github.com/llvm/circt/blob/3d19aa785468d4819961acf296c015d6f7fb318f/include/circt/Dialect/HW/HWStructure.td#L303

Reply:
InstanceOp has parent HWModuleOp, which in turn has parent ModuleOp, so I think ODS does generate a verifier to guarantee this.